bigcode_eval/generation.py

```diff
@@ -62,6 +63,43 @@ def parallel_generations(
                 )
         return generations[:n_tasks]
 
+    if args.modeltype == "wx":
+        wx_inference = WxInference()
+
+        return wx_inference.infer(
```

Comment:
Not sure how it works (I was testing with HF models) but it looks like `_parallel_generations()` returns nested loops `list[list[str]]` and the `infer()` gives back `list[str]`.